docs/settings/general-infra-logs-ui-settings.asciidoc

```diff
@@ -7,3 +7,6 @@ Controls the size of the composite aggregations used by the Inventory Threshold
 
 `xpack.infra.alerting.metric_threshold.group_by_page_size`::
 Controls the size of the composite aggregations used by the Metric Threshold group by feature. Defaults to `10_000`.
+
+`xpack.infra.logs.app_target`::
+Controls the target app for the logs client routing. Accepted values: `logs-ui | discover`. Defaults to `logs-ui`. 
```

Comment:
I appreciate the thought of adding documentation for this. But since this is supposed to be an internal setting for use in the serverless config, maybe we intentionally shouldn't document it?

Unfortunately there is no way to prevent its usage in non-serverless deployments right now, see https://github.com/elastic/kibana/issues/152395. But as soon as there is we probably want make use of it to restrict the usage of this new setting. Does that make sense?

Reply:
Totally, and thanks for pointing it out! 👌